Dockerfile

```diff
@@ -92,7 +92,7 @@ fi
 function get_dev_apt_deps() {
     if [[ "${DEV_APT_DEPS=}" == "" ]]; then
         DEV_APT_DEPS="apt-transport-https apt-utils build-essential ca-certificates dirmngr \
-freetds-bin freetds-dev git gosu krb5-user ldap-utils libffi-dev \
+freetds-bin freetds-dev git gosu graphviz graphviz-dev krb5-user ldap-utils libffi-dev \
```

Comment:
it's needed to install `eralchemy2`. It has only source packages but it links to graphviz C libraries and needs to be compiled. One of the reasons why adding it as depedency for "local" virtualenv might be a pain as it might not compile properly in all environments - and that's where having our shared CI image where we have those prerequisites is handy.

Reply:
BTW. Eralchemy2 is a fork of Eralchemy (which is not maintained any more but people needed to update it to support sqlalchemy 1.4 - that's why we use "2" - see https://github.com/Alexis-benoist/eralchemy/issues/94 - the power of open-source :D